pkg/cmd/flags/flags.go

```diff
@@ -0,0 +1,68 @@
+package flags
+
+import (
+	"strings"
+)
+
+// hasLeadingOrTrailingWhitespace checks if the string has leading or trailing whitespace.
+func hasLeadingOrTrailingWhitespace(s string) bool {
+	if strings.HasPrefix(s, " ") || strings.HasPrefix(s, "\t") ||
+		strings.HasSuffix(s, " ") || strings.HasSuffix(s, "\t") {
+		return true
+	}
+
+	return false
+}
+
+// isFlagOperatorValid checks if the operator is valid.
+func isFlagOperatorValid(operator string) bool {
+	switch operator {
+	case "=", "!=", "<", "<=", ">", ">=":
+		return true
+	default:
+		return false
+	}
+}
+
+type operatorAndValuesParts struct {
+	operatorAndValues string
+	operator          string
+	values            string
+}
+
+// getOperatorAndValuesParts splits a flag into its operator and values parts, validating them.
+// Valid formats: "xxxx=xxxx", "xxxx!=xxxx", "xxxx<xxxx", "xxxx<=xxxx", "xxxx>xxxx", "xxxx>=xxxx".
+//
+// It is used by parseScopeFlag and parseEventFlag.
+func getOperatorAndValuesParts(flag string, operatorIdx int) (operatorAndValuesParts, error) {
```

Comment:
Introduced getOperatorAndValuesParts() deduplicating logic between scope and event flags.